src/CMakeLists.txt

```diff
@@ -68,7 +76,8 @@ if (${Dawn_FOUND})
   target_link_libraries(libamber libamberenginedawn)
 endif()
 
-set(TEST_SRCS
+if (${AMBER_ENABLE_TESTS})
+  set(TEST_SRCS
     amberscript/parser_test.cc
```

Comment:
I think we must indent this line i.e., shift 2 white spaces to the right.

Reply:
Why? I'v been trying to indent them 2 spaces, not 4.

Reply:
Got it. I thought it would be nice if we indent it with `TEST_SRCS`.
Let's just keep the current one.